app/models/admin_set.rb

```diff
@@ -0,0 +1,3 @@
+class AdminSet < ActiveFedora::Base
```

Comment:
How are we going to ensure the Sufia version of AdminSet loads instead of the one in CurationConcerns?